cg/store/api/find_business_data.py

```diff
@@ -466,6 +467,7 @@ def get_samples_by_name_pattern(self, name_pattern: str) -> List[Sample]:
     def get_samples_by_customer_id_and_pattern(
         self, *, customers: Optional[List[Customer]] = None, enquiry: str = None
     ) -> List[Sample]:
+        """Get samples by customer and pattern."""
```

Comment:
It would be good to clarify which pattern? Sample name pattern? Consider renaming the parameter from enquiry to sample_name_pattern or something better.